desk/app/groups.hoon

```diff
@@ -83,7 +84,9 @@
           :+  %group-update       |  -:!>(*vale:m-group-update)
           :+  %group-log          |  -:!>(*vale:m-group-log)
         ::
-          :+  %group-token        &  -:!>(*vale:m-group-token)
```

Comment:
This should not have been strict in the first place, because tokens are only transferred internally for now.